Client/Application/AppDelegate.swift

```diff
@@ -131,6 +131,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         
         adjustIntegration = AdjustIntegration(profile: profile)
 
+        // We need to check if the app is a clean install to use for
+        // preventing the What's New URL from appearing.
+        if getProfile(application).prefs.intForKey(IntroViewControllerSeenProfileKey) == nil {
```

Comment:
How about using `"latestAppVersion"` here instead of `IntroViewControllerSeenProfileKey` ? I don't think this logic should be connected to the *Intro*. Better, create a `LatestAppVersionProfileKey`.